standard/types.md

```diff
@@ -9,10 +9,11 @@ type
     : reference_type
     | value_type
     | type_parameter
+    | pointer_type     // unsafe code support
     ;
 ```
 
-A third category of types, pointers, is available only in unsafe code [§23.3](unsafe-code.md#233-pointer-types).
+*pointer_type* is available only in unsafe code ([§23.3](unsafe-code.md#233-pointer-types)).
```

Comment:
Following previous wordsmithing suggestions, maybe:

> *pointer_type* ([§23.3](unsafe-code.md#233-pointer-types)) is available only in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).

There is a similar case below for *non_array_type* and maybe others.